prime-router/src/main/kotlin/Sender.kt

```diff
@@ -1,6 +1,8 @@
 package gov.cdc.prime.router
 
 import com.fasterxml.jackson.annotation.JsonIgnore
+import gov.cdc.prime.router.tokens.Jwk
+import gov.cdc.prime.router.tokens.JwkSet
```

Comment:
Per the FHIR standard, each Sender gives us one or more public keys, stored in a JWKSet.  Because permission for each scope can be turned on/off independently, we need to have a JWKSet for each scope (aka permission) that the Sender has been granted.  Note:  right now there's only one scope:  'report'.

In our implementation, for simplicity, we store these as part of our sender Settings.